src/Annotator/ExtraPropertyAnnotator.php

```diff
@@ -204,9 +204,29 @@ private function makeFirstAuthorDataItem() {
 	}
 
 	private function makeNumberOfPageViewsDataItem() {
-		if ( !$this->configuration['wgDisableCounters'] && $this->getWikiPage()->getCount() ) {
-			return new DINumber( $this->getWikiPage()->getCount() );
+		if ( $this->configuration['wgDisableCounters'] ) {
```

Comment:
No problem :-)

I just checked, and HitCounters does use the wgDisableCounters variable.